### DISCUSSION TITLE
Is Coqui itself not free for commercial use, or only the pretrained models it comes with?

### DISCUSSION THREAD

**Discussion by abhishek-junghare** [coqui-ai/TTS#4042] [General]
.

**Comment by manmay-nakhashi** [comment]
Only pretrained weights

**Comment by eginhard** [comment]
The Coqui toolkit is licensed [MPL 2.0](https://github.com/coqui-ai/TTS/blob/dev/LICENSE.txt), allowing commercial use. Licenses of pretrained models may be different and are listed here: https://github.com/coqui-ai/TTS/blob/dev/TTS/.models.json
